examples/app-playground/app/server-actions/NestedFormElement.tsx

```diff
@@ -0,0 +1,25 @@
+import Button from '#/ui/button';
+import { NameInput } from './components/NameInput';
+
+export function NestedFormElement() {
+  async function formAction(formData: FormData) {
+    'use server';
+    console.log('Nested Form Element Name: ', formData.get('name'));
```

Comment:
nit: will have double space

```suggestion
    console.log('Nested Form Element Name:', formData.get('name'));
```